nomination/templates/nomination/url_listing.html

```diff
@@ -1,222 +1,245 @@
-<script type="text/javascript" src="{% static 'nomination/scripts/core.js' %}"></script>
-<script type="text/javascript" src="{% static 'nomination/scripts/RelatedObjectLookups.js' %}"></script>
-
-<script type="text/javascript" src="{% static 'nomination/scripts/DateTimeShortcuts.js' %}"></script>
-<script type="text/javascript" src="{% static 'nomination/scripts/calendar.js' %}"></script>
-
-<link rel="stylesheet" href="{% static 'nomination/styles/jquery-ui.css' %}" type="text/css" media="all" />
-<script src="{% static 'nomination/scripts/jquery/jquery.min.js' %}" type="text/javascript"></script>
-<script src="{% static 'nomination/scripts/jquery/jquery-ui.min.js' %}" type="text/javascript"></script>
-<script src="{% static 'nomination/scripts/jquery/jquery.bgiframe-2.1.1.js' %}" type="text/javascript"></script>
-<script src="{% static 'nomination/scripts/jquery/jquery-ui-i18n.min.js' %}" type="text/javascript"></script>
-<script type="text/javascript" src="{% static 'nomination/scripts/nominationform.js' %}"></script>
-
-<!--<script src="http://maps.google.com/maps?file=api&amp;v=2&amp;key=ABQIAAAAiMxCv9HXrrInCAfSSpXWKRQFqlgjNlaZiz660GnjlAsHHd7dexRFxgdthjYu5L_NmdkGMwSomxrsnA"
-      type="text/javascript"></script>-->
-<script type="text/javascript">
-   $(document).ready(initForm);
-</script>
-<script type="text/javascript">
-    $.noConflict();
-    jQuery(document).ready(function($){
-        $('#institution-value').autocomplete({
-            source: {{ institutions|safe }},
-            delay: 0,
+{% block extra_headers %}
+
+    <script type="text/javascript">
+        var json_data = {{ json_data|safe }};
+        var form_types = {{ form_types|safe }};
+    </script>
+
+    <script type="text/javascript" src="{% static 'nomination/scripts/jquery.cookie.pack.js' %}"></script>
+    <script type="text/javascript" src="{% static 'nomination/scripts/core.js' %}"></script>
+    <script type="text/javascript" src="{% static 'nomination/scripts/RelatedObjectLookups.js' %}"></script>
+
+    <script type="text/javascript" src="{% static 'nomination/scripts/DateTimeShortcuts.js' %}"></script>
+    <script type="text/javascript" src="{% static 'nomination/scripts/calendar.js' %}"></script>
+
+    <link rel="stylesheet" href="{% static 'nomination/styles/jquery-ui.css' %}" type="text/css" media="all" />
+    <script src="{% static 'nomination/scripts/jquery/jquery.min.js' %}" type="text/javascript"></script>
+    <script src="{% static 'nomination/scripts/jquery/jquery-ui.min.js' %}" type="text/javascript"></script>
+    <script src="{% static 'nomination/scripts/jquery/jquery.bgiframe-2.1.1.js' %}" type="text/javascript"></script>
+    <script src="{% static 'nomination/scripts/jquery/jquery-ui-i18n.min.js' %}" type="text/javascript"></script>
+    <script type="text/javascript" src="{% static 'nomination/scripts/nominationform.js' %}"></script>
```

Comment:
I'll help you review these scripts. This is loading some scripts that are already in the header, as well as multiple versions of jquery.